microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,225 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+	* [CoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2ExperimentalObjectCollectionView interface](#)
+	* [CoreWebView2ExperimentalObjectCollectionView::get_Count method](#)
+	* [CoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalObjectCollectionView interface](#)
+	* [ICoreWebView2ExperimentalObjectCollectionView::get_Count method](#)
+	* [ICoreWebView2ExperimentalObjectCollectionView:: GetValueAtIndex method](#)
+
+---
+
+#### Promotions
+
+The following APIs are promoted to stable in this prerelease SDK:
+
+*  The SharedBuffer API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2_17 interface](#)
+	* [CoreWebView2_17::PostSharedBufferToScript method](#)
+* [CoreWebView2Environment12 interface](#)
+	* [ICoreWebView2Environment12:: CreateSharedBuffer method](#)
+* [CoreWebView2Frame4 interface](#)
+	* [CoreWebView2Frame4:: PostSharedBufferToScript method](#)
+* [CoreWebView2SharedBuffer interface](#)
+	* [CoreWebView2SharedBuffer:: OpenStream method](#)
+	* [CoreWebView2SharedBuffer:: Close method](#)
+	* [CoreWebView2SharedBuffer:: get_Size method](#)
+	* [CoreWebView2SharedBuffer:: get_Buffer method](#)
+	* [CoreWebView2SharedBuffer:: get_FileMappingHandle method](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2_17 interface](#)
+	* [CoreWebView2_17::PostSharedBufferToScript method](#)
+* [CoreWebView2Environment12 interface](#)
+	* [CoreWebView2Environment12:: CreateSharedBuffer method](#)
+* [CoreWebView2Frame4 interface](#)
+	* [CoreWebView2Frame4:: PostSharedBufferToScript method](#)
+* [CoreWebView2SharedBuffer interface](#)
+	* [CoreWebView2SharedBuffer:: OpenStream method](#)
+	* [CoreWebView2SharedBuffer:: Close method](#)
+	* [CoreWebView2SharedBuffer:: get_Size method](#)
+	* [CoreWebView2SharedBuffer:: get_Buffer method](#)
+	* [CoreWebView2SharedBuffer:: get_FileMappingHandle method](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2_17 interface](#)
+	* [ICoreWebView2_17::PostSharedBufferToScript method](#)
+* [ICoreWebView2Environment12 interface](#)
+	* [ICoreWebView2Environment12:: CreateSharedBuffer method](#)
+* [ICoreWebView2Frame4 interface](#)
+	* [ICoreWebView2Frame4:: PostSharedBufferToScript method](#)
+* [ICoreWebView2SharedBuffer interface](#)
+	* [ICoreWebView2SharedBuffer:: OpenStream method](#)
+	* [ICoreWebView2SharedBuffer:: Close method](#)
+	* [ICoreWebView2SharedBuffer:: get_Size method](#)
+	* [ICoreWebView2SharedBuffer:: get_Buffer method](#)
+	* [ICoreWebView2SharedBuffer:: get_FileMappingHandle method](#)
+---
+
+*  The Permission API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2Profile4 interface](#)
+	* [CoreWebView2Profile4::SetPermissionState method](#)
+	* [CoreWebView2Profile4::GetNonDefaultPermissionSettings method](#) 
+* [CoreWebView2PermissionSettingCollectionView interface](#)
```

Comment:
Are these applicable for .NET?  I omitted from my links.
* [CoreWebView2PermissionSettingCollectionView Class](#)
   * [CoreWebView2PermissionSettingCollectionView.GetValueAtIndex Method](#)
   * [CoreWebView2PermissionSettingCollectionView.Count Property](#)

Reply:
@peiche-jessica - is CoreWebView2PermissionSettingCollectionView Win32-only?

.NET latest prerelease has CoreWebView2PermissionSetting but not CoreWebView2PermissionSettingCollectionView.
WinRT latest prerelease: same.